deploy/kubeturbo-operator/helm-charts/kubeturbo/templates/configmap.yaml

```diff
@@ -55,4 +55,10 @@ data:
         "workloadController": "{{ default "" .Values.annotationWhitelist.workloadController }}"
       {{- end }}
       }
+    }
+  turbo-autoreload.config: |-
```

Comment:
Can you please add test results for 
1. Helm-based installs:  https://github.com/turbonomic/kubeturbo/wiki/Helm-Deployment-Details
2. Operator-based installs: https://github.com/turbonomic/kubeturbo/wiki/Operator-Details
3. Operatorhub installs: https://github.com/turbonomic/kubeturbo/wiki/OpenShift-Operator-Hub-Details

Reply:
@pankajbatra1 ,  the test result I put in the description is based on the operator. you can see I edited the CR in there. For the Helm-based installation, I did try it on my local. I can append the result later on. But for the Operatorhub installs,  I don't have a clear idea of how to test it.  I think I may have to create a new bundle and a test channel to publish it, after that, I think I can install the operator from UI to verify if it works or not.

Reply:
@pankajbatra1  Please check out my test case 3 in the description.   The new logging level field is shown on the page where the CR is created.   Thank @SumanthKodali999  for helping me publish my custom image in the beta channel.

Reply:
Regarding the helm install,  I tested it out and posted the result in another Kubeturbo [PR](https://github.com/turbonomic/kubeturbo/pull/900), please check the test case 3 for the detail